packages/wonder-blocks-dropdown/components/action-menu.test.js

```diff
@@ -212,6 +212,21 @@ describe("ActionMenu", () => {
         expect(menu.find(ActionItem).first()).toHaveProp("label", "Create");
     });
 
+    it("verifies testId is added to the opener", () => {
```

Comment:
Yay tests to catch this!

Reply:
Yes! and just in case, I'm adding tests to the other dropdowns as well (Single, Multi and Dropdown)